src/providers/cortex/trulens/providers/cortex/provider.py

```diff
@@ -142,14 +143,18 @@ def _invoke_cortex_complete(
 
         options = {"temperature": temperature}
 
-        completion_res_str: str = Complete(
+        completion_res: str = Complete(
             model=model,
             prompt=messages,
             options=options,
             session=self.snowpark_session,
             stream=False,
         )
-        return completion_res_str
+        # As per https://docs.snowflake.com/en/sql-reference/functions/complete-snowflake-cortex#returns,
```

Comment:
which version of `snowflake-ml-python` you are testing this implementation on?

Reply:
The returned value was a plain string (not stringnified json) at least in snowflake-ml-python>=1.7.1 and this would be failing for me

Reply:
Talked offline.